rpc/legacyrpc/methods.go

```diff
@@ -3104,6 +3108,141 @@ func signRawTransactions(s *Server, icmd interface{}) (interface{}, error) {
 	return &dcrjson.SignRawTransactionsResult{Results: toReturn}, nil
 }
 
+// startAutoBuyer handles the startautobuyer command.
+func startAutoBuyer(s *Server, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrjson.StartAutoBuyerCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, &ErrUnloadedWallet
+	}
+
+	config := &ticketbuyer.Config{
+		AccountName:             cmd.Account,
+		AvgPriceMode:            s.ticketbuyerConfig.AvgPriceMode,
+		AvgPriceVWAPDelta:       s.ticketbuyerConfig.AvgPriceVWAPDelta,
+		BlocksToAvg:             s.ticketbuyerConfig.BlocksToAvg,
+		DontWaitForTickets:      s.ticketbuyerConfig.DontWaitForTickets,
+		ExpiryDelta:             s.ticketbuyerConfig.ExpiryDelta,
+		FeeSource:               s.ticketbuyerConfig.FeeSource,
+		FeeTargetScaling:        s.ticketbuyerConfig.FeeTargetScaling,
+		MinFee:                  s.ticketbuyerConfig.MinFee,
+		MaxInMempool:            s.ticketbuyerConfig.MaxInMempool,
+		NoSpreadTicketPurchases: s.ticketbuyerConfig.NoSpreadTicketPurchases,
+		TxFee: s.ticketbuyerConfig.TxFee,
+	}
+
+	if cmd.BalanceToMaintain != nil {
+		if *cmd.BalanceToMaintain < 0 {
+			e := errors.New("Negative balance to maintain given")
```

Comment:
Try to use more descriptive error messages which tell the user/client what values must be used, instead of saying that a value is invalid, e.g. "balancetomaintain (-1.2345) must be non-negative".  Same goes for the rest of the errors below.